rmake.py

```diff
@@ -318,6 +318,10 @@ def config_cmd():
         raw_rocsolver_path = cmake_path(args.rocsolver_path)
         rocsolver_path_cmake =  f'"{raw_rocsolver_path}"'
 
+    raw_hipblas_common_path = cmake_path("C:/hipSDK")
+    hipblas_common_path = f'"{raw_hipblas_common_path}"'
```

Comment:
Yeah I guess would be good to add --hipblas_common_path cmd line argument. I can add it later though, doesn't have to hold up this PR.